examples/basic/android/app/src/main/java/com/videoplayer/MainApplication.java

```diff
@@ -1,51 +1,91 @@
 package com.videoplayer;
 
 import android.app.Application;
-
-import androidx.multidex.MultiDexApplication;
+import android.content.Context;
+import com.facebook.react.PackageList;
 import com.facebook.react.ReactApplication;
-import com.brentvatne.react.ReactVideoPackage;
+import com.facebook.react.ReactInstanceManager;
 import com.facebook.react.ReactNativeHost;
 import com.facebook.react.ReactPackage;
-import com.facebook.react.shell.MainReactPackage;
+import com.facebook.react.config.ReactFeatureFlags;
 import com.facebook.soloader.SoLoader;
-
-import com.reactnativecommunity.picker.RNCPickerPackage;
-
-import java.util.Arrays;
+import com.videoplayer.newarchitecture.MainApplicationReactNativeHost;
+import java.lang.reflect.InvocationTargetException;
 import java.util.List;
 
-public class MainApplication extends MultiDexApplication implements ReactApplication {
+public class MainApplication extends Application implements ReactApplication {
 
-  private final ReactNativeHost mReactNativeHost = new ReactNativeHost(this) {
-    @Override
-    public boolean getUseDeveloperSupport() {
-      return BuildConfig.DEBUG;
-    }
+  private final ReactNativeHost mReactNativeHost =
+      new ReactNativeHost(this) {
+        @Override
+        public boolean getUseDeveloperSupport() {
+          return BuildConfig.DEBUG;
+        }
 
-    @Override
-    protected List<ReactPackage> getPackages() {
-      return Arrays.asList(
-        new MainReactPackage(),
-        new ReactVideoPackage(),
-        new RNCPickerPackage()
-      );
-    }
+        @Override
+        protected List<ReactPackage> getPackages() {
+          @SuppressWarnings("UnnecessaryLocalVariable")
+          List<ReactPackage> packages = new PackageList(this).getPackages();
+          // Packages that cannot be autolinked yet can be added manually here, for example:
+          // packages.add(new MyReactNativePackage());
```

Comment:
Observation: (not a blocker) For some reason these two lines don't look like they are commented out? Just assuring that they are.

Reply:
Oh nvm! In the conversation view, they look commented out. 

Pls disregard this - This is what it looked like when I initially commented 
![Screen Shot 2023-06-06 at 1 45 11 PM](https://github.com/sleepio/react-native-video/assets/105252798/611ffd7a-8682-48da-8b63-3c7fa87a6b9e)